conf/nginx/st2.conf

```diff
@@ -36,6 +36,8 @@ server {
   ssl_certificate_key       /etc/ssl/st2/st2.key;
   ssl_session_cache         shared:SSL:10m;
   ssl_session_timeout       5m;
+  # If your version of OpenSSL supports it, you are strongly recommended to also (or only) use TLS v1.3
+  #ssl_protocols             TLSv1.2 TLSv1.3;
```

Comment:
This could be removed now, considering https://github.com/StackStorm/st2/pull/5280 is merged
```suggestion
```